### PR TITLE
[AF-323] Update iframe only template

### PR DESCRIPTION
**Pull request by danielbreves**
🍞

/cc @zendesk/vegemite

### Description
Update iframe only template to include new APIs

### References
* JIRA: https://zendesk.atlassian.net/browse/AF-323

### Risks
* None (template changes only)

### CODE REVIEW

**Comment by sandlerr** [comment]
wait for resize to hit prod?

**Comment by adammw** [comment]
:+1: for when API is working in production

File: app_template_iframe/assets/iframe.html [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-    var app = window.ZAFClient.init(function(context) {
-      var currentUser = context.currentUser;
-      console.log('Hi ' + currentUser.name);
-    });
-    app.postMessage('helloApp', { works: true });
+  // https://developer.zendesk.com/apps/docs/agent/zaf_v2
+    var client = ZAFClient.init();
+    client.invoke('resize', { width: '100%', height: '400px' });
   </script>
-  <p>It works!</p>
```
Comment (danielbreves) [inline app_template_iframe/assets/iframe.html]: Perhaps we should keep a "Hello world" here, just so it's not completely blank.

**Comment by svizzari** [comment]
👍

**Pull request merged**